job-runner/webapp/settings.json

```diff
@@ -7,6 +7,7 @@
   "limma_phenotype_export":"/tools/exporters/limma_phenotype_export.py",
   "gsea_path":"/tools/gsea/rgGSEA.py",
   "gsea_jar_path":"/tools/gsea/gsea2-2.2.2.jar",
+  "python":"python",
```

Comment:
I would say the better fix for this would have been to just hardcode python in the one place it's used because it’s a dictionary to itself, but that’s okay for now.